Makefile

```diff
@@ -157,7 +157,7 @@ lint-fix: fmt
 ## test: Run tests.
 test:
 	@echo "--> Running tests"
-	@go test -timeout 30m ./...
+	@go test -tags !nova -timeout 30m ./...
```

Comment:
maybe we can omit `!nova` here as it should default to false, and may just confuse people. wdyt?

Reply:
yep think we can remove